lib/vcv.ts

```diff
@@ -0,0 +1,162 @@
+import { useEffect, useState } from 'react'
+
+interface Entry {
+    str: string,
+    index: number,
+}
+
+interface Diff {
+    added: Entry[],
+    removed: Entry[],
+}
+
+function sumEntries(entries: Entry[]): Entry[] {
+    const result: Entry[] = []
+    entries = entries.sort((a, b) => a.index - b.index)
+    let offset = 1
+    for (let i = 0; i < entries.length; i++){
+        if (result.length === 0){
+            result.push(entries[i])
+            offset = 1
+        } else if (entries[i].index === result[result.length - 1].index + offset){
```

Comment:
I am not sure what's the purpose of this commits history, you can keep if it you want. But please refactor this code and drop `console.logs` and comments